Make.ps1

```diff
@@ -83,6 +83,7 @@ param (
 )
 
 function Init {
+    pip install -e '.[pre-dev]'
```

Comment:
curious about this change. I think our Makefile doesn't install the pre-dev ones: https://github.com/aws/aws-sam-cli/blob/e0d02edad389e4933fb550144ed9e9e3cc81e5db/Makefile#L6

Reply:
I don't believe we have as many windows devs, so this might have gotten missed out, so I brought back the pre-dev step in there.